openshift/unleash.yaml

```diff
@@ -178,3 +180,4 @@ parameters:
   value: app-sre
 - name: team
   value: app-sre-cluster
+- name: read_only_team
```

Comment:
This param is mandatory? do we want o be backward compatible, supporting only an admin group ?

Reply:
Backwards compatibility was the plan but it seems I missed this in the template. Adding an empty default value should be sufficient?

Reply:
I think so. not quite sure what this does in js:
```
if (READ_ONLY_TEAM && ...
```

Reply:
in javascript a value is `truthy` if it is not 
* null
* undefined
* NaN
* empty string ("")
* 0
* false

this behaviour is also used here
https://github.com/app-sre/unleash/blob/88301b349506ca4d79a54eeda1791baa9eb7cafb/index.js#L23-L25

Reply:
i tested the template behaviour. a missing default value `value` on a parameter translates to an empty string when interpolated in the template. so this is already backwards compatible